src/validations/rules/sar.sch

```diff
@@ -342,6 +338,62 @@
                     else
                         true()">A risk with a status of 'closed' must have a risk-log/entry with a status-change of
                 'closed'.</sch:assert>
+                
+                <sch:assert
+                diagnostics="has-duplicate-priority-value-diagnostic"
+                fedramp:specific="true"
+                id="has-duplicate-priority-value"
+                role="error"
+                test="
+                    if (oscal:prop[@name = 'priority'])
+                    then
+                        if (normalize-space($risk-priority-values) = '')
+                        then
+                            true()
+                        else
+                            if (oscal:prop[@name = 'priority']/@value[. ne $risk-priority-values])
+                            then
+                                true()
+                            else
+                                false()
+                    else
+                        true()">Risks with priority properties must have unique priority values.</sch:assert>
+        </sch:rule>
+        
+        <sch:let
+            name="risk-priority-values"
+            value="distinct-values(//oscal:risk/oscal:prop[@name = 'priority']/@value[. = following::oscal:risk/oscal:prop[@name = 'priority']/@value])" />
+
+        <sch:rule
+            context="oscal:result">
+            <sch:assert
+                diagnostics="has-attestation-diagnostic"
+                fedramp:specific="true"
+                id="has-attestation"
+                role="error"
+                test="oscal:attestation[oscal:part[@name = 'authorization-statements']/oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'recommend-authorization']]">
+                There must exist an attestation with a part containing a property with a name of 'recommend-authorization'.</sch:assert>
+        </sch:rule>
+        
+        <sch:rule
+            context="oscal:attestation/oscal:part[@name = 'authorization-statements'][oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'recommend-authorization']]">
+            <sch:assert
+                diagnostics="has-attestation-value-no-diagnostic"
+                fedramp:specific="true"
+                id="has-attestation-value-no"
+                role="error"
+                see="Guide to OSCAL-based FedRAMP Security Assessment Reports - Section 4.12"
+                test="
+                    if (oscal:prop/@value ne 'yes')
+                    then
+                        if (matches(normalize-space(oscal:part[1]/oscal:p[1]), 'A total of .* system risk(s?) were identified for .*, including .* High risk(s?), .* Moderate risk(s?), .* Low risk(s?), and .* of operationally required risk(s?).'))
```

Comment:
`.*` has cardinality {0,}; is that correct (i.e., nothing but two spaces in those locations is acceptable).
If these are going to be numbers as opposed to anything, try using `\d+`.

Reply:
I didn't feel confident from the Guide narrative that the value would always be a digit.  I thought of [a-zA-Z0-9]* but also wasn't sure if that was overkill.

I'll adjust to add the alpha and digit instead.

Reply:
2nd thought.  Went with \w+ to accommodate the digits and alphas as well as state that there couldn't be empty space.